front/src/modules/ui/components/buttons/Button.tsx

```diff
@@ -130,23 +147,47 @@ const StyledButton = styled.button<
   }
 `;
 
+const SoonPill = styled.span`
+  align-items: center;
+  background: ${({ theme }) => theme.background.transparent.light};
+  border-radius: 50px;
+  color: ${({ theme }) => theme.font.color.light};
+  display: flex;
+  font-family: Inter;
+  font-size: 11px;
+  font-style: normal;
+  font-weight: 500;
+  gap: 8px;
```

Comment:
@FelixMalfait Same here for the `gap` and `padding`. `theme.spacing` helper should be used